src/aws/helpers.ts

```
@@ -329,19 +345,167 @@ export const resizeAndSaveVariants = async ({
+export const getGroupedTagsByPlayer = (
+  groupedImages: S3ImageMeta[][] = [],
+  appendToTagData = {}
+) => {
+  if (!groupedImages.length) return []
+
+  const playerGroupedImages: Tag[] = []
+  const playerGroupedTags: any[] = []
+
+  // Determine the highest tagnumber (assumes array index = tagnumber)
+  const highestTagnumber = groupedImages.reduce((max, group, index) => {
+    return group && group.length ? Math.max(max, index) : max
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)